week1/index.js

```diff
@@ -1,38 +1,44 @@
-import HTTP from 'http'
-import Path from 'path'
-
-import sendIndexHTML from './responses/sendIndexHTML'
-import sendPage2HTML from './responses/sendPage2HTML'
-import sendStylesCSS from './responses/sendStylesCSS'
-import sendText from './responses/sendText'
-
-const server = HTTP.createServer((request, response) => {
-	console.log(request.method, request.url)
-
-	switch (request.url) {
-	case '/':
-		sendIndexHTML(response)
-		break
-	case '/page2':
-		sendPage2HTML(response)
-		break
-	case '/styles.css':
-		sendStylesCSS(response)
-		break
-	default:
-		const extension = Path.extname(request.url)
-		if (extension === '') {
-			response.statusCode = 302
-			response.setHeader('Location', '/')
-		} else {
-			response.statusCode = 404
-			sendText(response, "File not found")
-		}
-	}
-	
-	response.end()
-})
-
-server.listen(3001)
-
-console.log('Server started')
\ No newline at end of file
+//import HTTP from 'http'
+//import Path from 'path'
+//
+//import sendIndexHTML from './responses/sendIndexHTML'
+//import sendPage2HTML from './responses/sendPage2HTML'
+//import sendStylesCSS from './responses/sendStylesCSS'
+//import sendText from './responses/sendText'
+
+const http = require('http');
+const port = 3000
+let state = 10;
+
+
+const server = http.createServer( (request, response) => {
+
+    console.log(`I have received a request for path: ${ request.url }`);
+
+    
+    
+    if ( request.url === "/" ) {
+        response.write("You requested root path.");
+    } else if ( request.url === "/state" ) {
+        response.write(`<h1>${state}</h1>`);
+         
+    }else if ( request.url === "/add" ) {
+        state++;
+        response.write(`<h1>${state}</h1>`);
+    }else if ( request.url === "/remove" ) {
+        state--;
+        response.write(`<h1>${state}</h1>`);
+    }else if ( request.url === "/reset" ) {
+        state = 10;
+        response.write(`<h1>${state}</h1>`);
+    }else {
+        response.write(`return error code 404: Not found with a friendly message`)        
```

Comment:
Since you haven't set an http status code, this will probably return 200 (the default). In the case of a 'not found', we probably want to return with a status code of 404 - can you make this change? Check the Node HTTP documentation for details.

Also, you can add your own friendly message!